parameter/ParameterType.cpp

```diff
@@ -100,10 +99,9 @@ void CParameterType::setXmlUnitAttribute(CXmlElement& xmlElement) const
 
 // XML Serialization value space handling
 // Value space handling for configuration import/export
-void CParameterType::handleValueSpaceAttribute(CXmlElement& xmlConfigurableElementSettingsElement, CConfigurationAccessContext& configurationAccessContext) const
+void CParameterType::handleValueSpaceAttribute(CXmlElement& /*xmlConfigurableElementSettingsElement*/,
```

Comment:
I do not agree `elem` has less information than `xmlConfigurableElementSettingsElement`